service/api_logic/user_logic.py

```diff
@@ -0,0 +1,96 @@
+from flask import current_app, url_for
+from flask_mail import Message
+from itsdangerous import URLSafeTimedSerializer
+from dto.api_output import OutPutUser, OutputPreferences
+from database.models import User
+from exept.handle_exeptions import handle_exceptions
+import bcrypt
+from logger.logger import Logger
+from jinja2 import Environment, FileSystemLoader
+import os
+from flask_jwt_extended import create_access_token
+
+api_logic_logger = Logger("api_logic_logger", "api_logic_logger.log")
```

Comment:
Don't like the idea of having logger (BTW - change the name to either just "log" or "logger") here. It cannot exist without UserService, right? Then it should be within the said service (Singleton)